mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -112,7 +112,7 @@ view: +subscriptions {
   dimension: cancel_reason {
     type: string
     sql: CASE
-      WHEN ${provider} LIKE "Apple Store IAP" THEN "Cancelled by IAP"
```

Comment:
Should this still be named as "Cancelled by IAP"?

Reply:
we could rename it to `Cancelled by Apple`, but that will break dashboards